Makefile

```diff
@@ -39,7 +39,7 @@ vet: | test
 	go vet $(MODULE)/...
 
 check: ## Run tests and CI checks
-check: test test-race vet gofmt staticcheck misspell unconvert unparam ineffassign yamllint check-stale
+check: test test-race vet gofmt staticcheck misspell unconvert unparam ineffassign yamllint check-stale site-check
```

Comment:
My workflow is normally `make check`, but the Docker install takes a long time. I can switch to `make test` but not sure whether we want `check` to take longer by default.